kbase-extension/static/errorMain.js

```diff
@@ -21,7 +21,7 @@ require([
             return Login.init($('#signin-button'), true);
         })
         .then(function () {
-            let statusCode = document.getElementsByClassName('error')[0].getAttribute('data-code');
+            let statusCode = document.getElementById('error_container').getAttribute('data-code');
```

Comment:
accessing by the element ID instead of the class, since I changed the class

Reply:
statusCode doesn't change, so it can be const.